sources/stat.h

```diff
@@ -20,14 +21,16 @@ struct od_stat_state
 
 struct od_stat
 {
+	bool enable_quantiles;
+	uint8_t current_tdigest;
 	od_atomic_u64_t count_query;
 	od_atomic_u64_t count_tx;
 	od_atomic_u64_t query_time;
 	od_atomic_u64_t tx_time;
 	od_atomic_u64_t recv_server;
 	od_atomic_u64_t recv_client;
-	od_hgram_t *transaction_hgram;
-	od_hgram_t *query_hgram;
+	td_histogram_t* transaction_hgram[5];
+    td_histogram_t* query_hgram[5];
```

Comment:
Why 5?

Reply:
That was in start version. I thought that 5 minutes is pretty ok. I can replace it with setting from config.